R/control.ergm.R

```diff
@@ -460,26 +477,30 @@
 #' @keywords models
 #' @export control.ergm
 control.ergm<-function(drop=TRUE,
-
+                       
                        init=NULL,
                        init.method=NULL,
                        
                        main.method=c("MCMLE", "Stochastic-Approximation"),
                        force.main=FALSE,
                        main.hessian=TRUE,
-
+                       
                        checkpoint=NULL,
                        resume=NULL,
-
+                       
                        MPLE.samplesize=.Machine$integer.max,
                        init.MPLE.samplesize=function(d,e) max(sqrt(d),e,40)*8,
                        MPLE.type=c("glm", "penalized","logitreg"),
                        MPLE.maxit=10000,
                        MPLE.nonvar=c("warning","message","error"),
                        MPLE.nonident=c("warning","message","error"),
                        MPLE.nonident.tol=1e-10,
+                       MPLE.covariance.samplesize =500,
+                       MPLE.covariance.method ="invHess",
+                       MPLE.covariance.sim.burnin = 1028,
+                       MPLE.covariance.sim.interval = 1028,
```

Comment:
Why these values?

Reply:
There is no particular reason, but these values feed into simulate.ergm(). So I set the burnin and the interval equal to the default burnin and interval values in control.simulate.R

Reply:
> There is no particular reason, but these values feed into simulate.ergm(). So I set the burnin and the interval equal to the default burnin and interval values in control.simulate.R

I believe those are 2^10 = 1024.